### PR TITLE
Create AV Jiali scraper

### PR DESCRIPTION
**Pull request by smcallah**
Noticed work on an AV Jiali scraper stalled out over 2 years ago.

Created this from the Tenshiago.yml scraper as the sites are very similar, but there are some code differences for performer name and tags that caused the Tenshiago scraper to not quite work with avjiali.com

### CODE REVIEW

File: scrapers/AVJiali.yml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+              - regex: (\d+)(st|nd|rd|th)
+                with: "$1"
+          - parseDate: January 02, 2006
+      Performers:
+        Name: //div[@class="cat"][1]/a[starts-with(@href, 'https://avjiali.com/model')]
+        URL: //div[@class="cat"][1]/a/@href
+      Tags:
+        Name: //div[@class="cat"][2]/a
+      Details:
+        selector: //h1[@class="none"]
```
Comment (Maista6969) [inline scrapers/AVJiali.yml:30]: This is just grabbing the title again, I'd suggest the following to grab the full details
```
Details:
  selector: //p[span[@class="readmore"]]
  postProcess:
    - replace:
        - regex: ... Read More
          with:
```

File: scrapers/AVJiali.yml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+      URL: //link[@rel="canonical"][1]/@href
+      Image:
+        selector: //img[@class="rounded"]/@src
+        postProcess:
+          - replace:
+              - regex: ^//
+                with: https://
+              - regex: 160x160
+                with: 500x500
+# Last Updated October 2, 2023
```
Comment (Maista6969) [inline scrapers/AVJiali.yml:138]: The validator is incredibly picky, this needs to be `October 02` and not just `October 2` 😅

**Review by Maista6969 — CHANGES_REQUESTED** [review]
Looks good! Once this passes validation I can merge it and we can close #1315 as well

File: scrapers/AVJiali.yml (hunk 1 above)
Comment (smcallah) [reply scrapers/AVJiali.yml:30]: Was just working on this.  The description is in 3 different tags, so I'm having trouble with concatenating it all together.

File: scrapers/AVJiali.yml (hunk 2 above)
Comment (smcallah) [reply scrapers/AVJiali.yml]: Thanks, I almost put the 0.

File: scrapers/AVJiali.yml (hunk 1 above)
Comment (Maista6969) [reply scrapers/AVJiali.yml:30]: I was looking at the same thing (tried with the `string()` xpath function) but I realized if we just grab it from the `<p>` and remove the `... Read More` bit it works great, did you try the selector I suggested?

**Comment by Maista6969** [comment]
This also needs an entry in `SCRAPERS-LIST.md` 🙂

File: scrapers/AVJiali.yml (hunk 1 above)
Comment (smcallah) [reply scrapers/AVJiali.yml]: Thanks, worked great.  I also fixed an issue with the Details: under the performer scraper copied from the tenshigao.yml.

**Comment by Maista6969** [comment]
Thank you for your contribution 🙂

**Review by Maista6969 — APPROVED** [review]

**Pull request merged**